app/uk/gov/hmrc/icedsubscriptionfrontend/views/SuccessfullyEnrolledPage.scala.html

```diff
@@ -34,14 +34,16 @@
 
     @paragraph("successfullyEnrolled.para.1")
 
-    @paragraph("successfullyEnrolled.coms")
+    @paragraph("successfullyEnrolled.para.1")
+
+    @hmrcPageHeading(PageHeading(text = messages("successfullyEnrolled.heading")))
```

Comment:
this should be a h2 not a page heading/h1
see here for an example https://github.com/hmrc/iced-subscription-frontend/blob/master/app/uk/gov/hmrc/icedsubscriptionfrontend/views/LandingPage.scala.html#L62
Suggest also wrapping the section in a div like shown on the landing page